app/src/main/res/values/strings.xml

```diff
@@ -646,4 +646,6 @@
     <string name="feed_use_dedicated_fetch_method_disable_button">Disable fast mode</string>
     <string name="feed_use_dedicated_fetch_method_help_text">Do you think feed loading is too slow? If so, try enabling fast loading (you can change it in settings or by pressing the button below).\n\nNewPipe offers two feed loading strategies:\n• Fetching the whole subscription channel, which is slow but complete.\n• Using a dedicated service endpoint, which is fast but usually not complete.\n\nThe difference between the two is that the fast one usually lacks some information, like the item\'s duration or type (can\'t distinguish between live videos and normal ones) and it may return less items.\n\nYouTube is an example of a service that offers this fast method with its RSS feed.\n\nSo the choice boils down to what you prefer: speed or precise information.</string>
     <string name="content_not_supported">This content is not yet supported by NewPipe.\n\nIt will hopefully be supported in a future version.</string>
+    <string name="search_showing_result_for">Showing results for</string>
+    <string name="search_did_you_mean">Did you mean:</string>
```

Comment:
you shouldn't do like that, format using `%s` or `%1$s`: some languages could be like: 
**newpipe**: did you mean, or 
**newpipe** showing results for.

Reply:
That's true indeed, however how will you do the differently styled suggestion then?

Reply:
Yeah, but I think this would happen specifically in languages that are RTL (like Hebrew or Arabic), and in this case I assume the whole layout is mirrored to RTL (and at least for Hebrew and Arabic, I know this is the way it's supposed to be).

Reply:
@wb9688 
1. Using Markwon, a markdown library we already use, and surround the word in with `**` (bold in markdown), like: `Showing results for **%1$s**`. See VideoDetailFragment#prepareDescription() for an usage. It's the easiest solution.
2. Using spannable, see https://stackoverflow.com/questions/20936901/combining-spannable-with-string-format . But it seems harder, and not better.

@Royosef It's false, go in Settings -> Content -> App language, pick Arabic, the second language (actually the first after system default), you'll see nothing is changed except text. As it's two different textviews, it won't work as you think, and that's why you need to format.

By the way
```suggestion
```
edit the already existing did_you_mean string, it's unused.

Reply:
@B0pol: Not really a good idea, as you could have service return weird characters that would make the formatting weird. In this case I'd probably choose for just having e.g. `Did you mean "%s"?` and `Showing results for "%s"` with normal formatting.

Reply:
So, @ B0pol, are you cool with a plain string format?

Reply:
I don't see how like that could happen. But yes